fleet_booking/doc/index.rst

```diff
@@ -0,0 +1,135 @@
+===============================
+ Custom system for car renting
+===============================
+
+It uses (inherits) HR and Fleet odoo 9 genuine modules.
+
+How to check it:
+
+* Install module
+* Go to fleet in main menu
+* Go to department in Configuration section
+
+Updates is here: https://github.com/yelizariev/addons-dev/pull/57
+
+
+==============
+Specifications
+==============
+
+Customer features
+-----------------
+
+1 Stage::
+
+   * Create or edit customer
+   * User with Branch Officer role enters in database.
+   * Open Contacts.
+   * Open some contact.
+   * You will see customer form.
+   * Press edit button.
+   * Enter Date of Birth.
+   * If customer age less than 21 you will not be able to save contact and you will see according notification.
+   * Enter Nationality string.
```

Comment:
Consider to use this module
https://github.com/ingadhoc/partner/blob/8.0/partner_person/__openerp__.py